core/jazz_environment-event-handler/index.js

```diff
@@ -128,6 +128,7 @@ function processEachEvent(record, configData, authToken) {
 
 function checkForInterestedEvents(encodedPayload, sequenceNumber, config) {
   return new Promise((resolve, reject) => {
+    logger.info("JSON parse 1", new Buffer(encodedPayload, 'base64').toString('ascii'));
```

Comment:
logger.debug

Reply:
sure, I will remove all these logs when the feature is ready.